collated/main/A0141006B.md

```diff
@@ -0,0 +1,1280 @@
+            calendarDate = calendarDate.minusDays(1);
+        }
+        // Populate the calendar with day numbers
+        for (AnchorPaneNode ap : allCalendarDays) {
+            if (ap.getChildren().size() != 0) {
+                ap.getChildren().remove(0);
+            }
+            Text txt = new Text(String.valueOf(calendarDate.getDayOfMonth()));
+            ap.setDate(calendarDate);
+            ap.setTopAnchor(txt, 3.0);
+            ap.setLeftAnchor(txt, 3.0);
+            ap.getChildren().add(txt);
+            calendarDate = calendarDate.plusDays(1);
+        }
+        // Change the title of the calendar
+        calendarTitle.setText(yearMonth.getMonth().toString() + " " + String.valueOf(yearMonth.getYear()));
+    }
+
+    /**
+     * Move the month back by one. Repopulate the calendar with the correct dates.
+     */
+    private void previousMonth() {
+        currentYearMonth = currentYearMonth.minusMonths(1);
+        populateCalendar(currentYearMonth);
+    }
+
+    /**
+     * Move the month forward by one. Repopulate the calendar with the correct dates.
+     */
+    private void nextMonth() {
+        currentYearMonth = currentYearMonth.plusMonths(1);
+        populateCalendar(currentYearMonth);
+    }
+
+    public VBox getView() {
+        return view;
+    }
+
+    public ArrayList<AnchorPaneNode> getAllCalendarDays() {
+        return allCalendarDays;
+    }
+
+    public void setAllCalendarDays(ArrayList<AnchorPaneNode> allCalendarDays) {
+        this.allCalendarDays = allCalendarDays;
+    }
+}
+```
+###### /java/seedu/cmdo/ui/HelpCard.java
+``` java
+ *
+ */
+
+public class HelpCard extends UiPart{
+
+	private static final String FXML = "HelpCard.fxml";
+	
+	public String commandName;
+	public String commandDescription;
+	public String commandSyntax;
+	
+	@FXML
+	private ScrollPane helpCardScrollPane;
+	@FXML
+	private VBox helpCardPane;
+	@FXML
+	private AnchorPane testPane;
+	
+	@FXML
+	private VBox addCard;
+	@FXML
+	private VBox blockCard;
+	@FXML
+	private VBox clearCard;
+	@FXML
+	private VBox deleteCard;
+	@FXML
+	private VBox doneCard;
+	@FXML
+	private VBox editCard;
+	@FXML
+	private VBox exitCard;
+	@FXML
+	private VBox findCard;
+	@FXML
+	private VBox helpCard;
+	@FXML
+	private VBox listCard;
+	@FXML
+	private VBox selectCard;
+	@FXML
+	private VBox storageCard;
+	
+	@FXML
+	private Label addNameLabel;
+	@FXML
+	private Label addDescriptionLabel;
+	@FXML
+	private Label blockNameLabel;
+	@FXML
+	private Label blockDescriptionLabel;
+	@FXML
+	private Label clearNameLabel;
+	@FXML
+	private Label clearDescriptionLabel;
+	@FXML
+	private Label deleteNameLabel;
+	@FXML
+	private Label deleteDescriptionLabel;
+	@FXML
+	private Label doneNameLabel;
+	@FXML
+	private Label doneDescriptionLabel;
+	@FXML
+	private Label editNameLabel;
+	@FXML
+	private Label editDescriptionLabel;
+	@FXML
+	private Label exitNameLabel;
+	@FXML
+	private Label exitDescriptionLabel;
+	@FXML
```

Comment:
\O/ i'm sure you can combine the declarations together doing this doesn't increase your LoC, infact you might be penalised